src/components/common/SuperMaterialTable.tsx

```diff
@@ -62,6 +79,17 @@ function SuperMaterialTable<Entry extends EntryID>({
     props.options.searchText = props.urlQueryParams.search || undefined;
   }
 
+  const { columns } = props;
```

Comment:
I think we should be consistent, if already use destructuring for the column, we should use the same for e.g. `urlQueryParams`.

Reply:
Yes that is a good idea I will try to extract all the properties from props at once and then use them.